ui/raidboss/data/05-shb/raid/e5s.js

```diff
@@ -55,15 +55,15 @@
       regexJa: Regexes.startsUsing({ id: '4BAC', source: 'ラムウ', capture: false }),
       regexKo: Regexes.startsUsing({ id: '4BAC', source: '라무', capture: false }),
       condition: function(data) {
-        return !data.firstAdd;
```

Comment:
What's wrong with this? I think the code as written should be fine, as !undefined is true.

Reply:
ummmm, that's right. But why did this mis-functioned... It showed message on first add...

Reply:
Oh! You *don't* want it to run the first time? I would remove the condition then, and just have a 
```
infoText: function(data) {
  if (data.seenFirstAdd) {
    return { look for adds etc };
  }
  data.seenFirstAdd = true;
}
```